Fix spelling-error-in-binary Commiting Committing in DejavuPredictor.
Index: presage-0.8.3/src/lib/predictors/dejavuPredictor.cpp
===================================================================
--- presage-0.8.3.orig/src/lib/predictors/dejavuPredictor.cpp	2010-06-14 10:12:35.000000000 +0100
+++ presage-0.8.3/src/lib/predictors/dejavuPredictor.cpp	2010-06-14 10:12:48.000000000 +0100
@@ -117,7 +117,7 @@
 	 it++)
     {
 	std::string new_token = *it;
-	logger << INFO << "Commiting new token to memory: " << new_token << endl;
+	logger << INFO << "Committing new token to memory: " << new_token << endl;
 	std::ofstream memory_file(memory.c_str(), std::ios::app);
 	if (!memory_file) {
 	    logger << ERROR << "Error opening memory file: " << memory << endl;
