Fix lintian complaint shlib-calls-exit.

Index: presage-0.8.3/src/lib/predictors/smoothedCountPredictor.cpp
===================================================================
--- presage-0.8.3.orig/src/lib/predictors/smoothedCountPredictor.cpp	2010-06-13 18:26:26.000000000 +0100
+++ presage-0.8.3/src/lib/predictors/smoothedCountPredictor.cpp	2010-06-13 18:27:04.000000000 +0100
@@ -305,7 +305,6 @@
 		    } else {
 			std::cerr << "Invalid invocation of buildPrediction method!"
 				  << std::endl;
-			exit( 1 );
 		    }
 		}
 		return 0;
