Enable usage of system libtinyxml.

Index: presage-0.8.2/configure.ac
===================================================================
--- presage-0.8.2.orig/configure.ac	2010-06-10 23:05:22.000000000 +0100
+++ presage-0.8.2/configure.ac	2010-06-10 23:05:31.000000000 +0100
@@ -38,6 +38,8 @@
 AC_PROG_CPP
 AM_PROG_CC_C_O
 
+AC_LANG([C++])
+
 dnl ==================
 dnl Checks for libtool
 dnl ==================
@@ -79,6 +81,21 @@
                  [])
 AM_CONDITIONAL(HAVE_CURSES, test "x$have_curses_library" = "xtrue" -a "x$have_curses_header" = "xtrue")
 
+
+dnl ==================
+dnl Checks for tinyxml
+dnl ==================
+AC_CHECK_LIB([tinyxml],
+             [main],
+             [have_tinyxml_library=true],
+	     [AC_MSG_WARN([tinyxml library not found. Embedded tinyxml library will be built.])])
+AC_CHECK_HEADERS([tinyxml.h],
+                 [have_tinyxml_header=true],
+                 [AC_MSG_WARN([tinyxml library not found. Embedded tinyxml library will be built.])],
+                 [])
+AM_CONDITIONAL(BUILD_TINYXML, test "x$have_tinyxml_library" != "xtrue" -o "x$have_tinyxml_header" != "xtrue")
+
+
 dnl =================
 dnl Checks for SQLite
 dnl =================
Index: presage-0.8.2/src/lib/Makefile.am
===================================================================
--- presage-0.8.2.orig/src/lib/Makefile.am	2010-06-10 23:05:17.000000000 +0100
+++ presage-0.8.2/src/lib/Makefile.am	2010-06-10 23:05:31.000000000 +0100
@@ -21,7 +21,11 @@
 
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = 			tinyxml core simulator predictors #plump
+if BUILD_TINYXML
+MAYBE_TINYXML =	tinyxml
+endif
+
+SUBDIRS =	$(MAYBE_TINYXML) core simulator predictors
 
 include_HEADERS =		presage.h \
 				presageException.h \
@@ -33,7 +37,6 @@
 				presageException.h \
 				presageCallback.h 
 libpresage_la_LIBADD =		core/libcore.la \
-				tinyxml/libtinyxml.la \
 				simulator/libsimulator.la \
 				predictors/libpredictor.la  \
 				predictors/libSmoothedNgramPredictor.la \
@@ -45,11 +48,9 @@
 				predictors/libDejavuPredictor.la \
 				predictors/libARPAPredictor.la 
 
-#				REVISIT remove predictors/lib*.la entries
-#				when reintegrating dynamically loaded predictors
-
-#				REVISIT add libplump.la when using plump
-#				plump/src/libplump.la 
+if BUILD_TINYXML
+libpresage_la_LIBADD +=		tinyxml/libtinyxml.la
+endif
 
 libpresage_la_LDFLAGS =		-no-undefined \
 				-version-info 2:1:1
